### PR TITLE
Updated the images with the new project selector

### PR DESCRIPTION
**Pull request by marcalcobe**
Updated the images with the new project selector and description texts.

### CODE REVIEW

**Comment by marcalcobe** [comment]
https://community.openproject.org/work_packages/41948/activity

File: docs/getting-started/projects/README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ To configure further project information, see the documentation for [project set
 
 ### View all projects
 
-To view all your projects in which you are a member, click on the **View all projects** link in the **Select a project** drop-down menu.
+To view all your projects in which you are a member, click on the **Projects list** button in the action bar after opening the **Select a project** drop-down menu.
 
 ![view all projects](view_all_projects.png)
```
Comment (birthe) [inline docs/getting-started/projects/README.md:75]: This screenshot, showing only the button, looks a bit irritating for users who don't know the application. I would rather show the whole menu, including some projects and the button at the bottom.

**Review by birthe — APPROVED** [review]
Hi @marcalcobe I quickly updated the one screenshot.

**Comment by birthe** [comment]
Hi @marcalcobe here are more places with the old screens:
https://www.openproject.org/docs/user-guide/projects/#project-structure
https://www.openproject.org/docs/user-guide/projects/#global-projects-overview---view-all-projects
I have the PR open, so I'll quickly add the changes myself.

**Comment by birthe** [comment]
Hi @as-op, I have changed the following headlines:
Old: ## Global projects overview - View all projects
New: ## Projects list
I am pretty sure we do have links to the old headline. Could you please check this for me? Thank you

File: docs/getting-started/projects/README.md (hunk 1 above)
Comment (marcalcobe) [reply docs/getting-started/projects/README.md:75]: Hi @birthe,
Make sense to me. Thanks for the update.

**Review by as-op — APPROVED** [review]

**Pull request merged**